lib/velocity-animate-shim.js

```diff
@@ -2,16 +2,17 @@
 // requires window. Note that this just no-ops the components so
 // that they'll render, rather than doing something clever like
 // statically rendering the end state of any provided animations.
-if (typeof window !== 'undefined') {
+if ((process && process.env && process.env.NODE_ENV === 'test') ||  typeof window === 'undefined') {
```

Comment:
If `process` is undefined then this will error out, rather than just be false. You need to use `typeof` to check, the same way `window` is checked.